bin/generate-podspecs.sh

```diff
@@ -17,8 +17,8 @@ function warn_missing_tag_commit() {
 }
 
 # Change to the expected directory.
-cd "$( dirname "$0" )"
-cd ..
+pushd "$( dirname "$0" )" > /dev/null
+popd > /dev/null
```

Comment:
Love the use of `pushd` and `popd`. The output redirect to `/dev/null` is a bit cumbersome to do on every usage, though.

We _could_ change the shebang to run on ZSH, and use `setopt PUSHDSILENT` at the start of the script. What do you think? 

Not a blocker! Just a suggestion.